<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BresenhamClass</class>
 <widget class="QWidget" name="BresenhamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bresenham</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>12</y>
      <width>94</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>94</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>antialiasing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>62</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Line</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>581</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Bresenham.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clear_btn</sender>
   <signal>clicked()</signal>
   <receiver>BresenhamClass</receiver>
   <slot>clear_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clear_slot()</slot>
 </slots>
</ui>
